### ISSUE TITLE
[MSYS2] Synfig CLI hangs when rendering about_icon.sif

### ISSUE THREAD

**Issue by morevnaproject** [synfig/synfig#861]
**Synfig version & platform**:
Master branch, commit id dbcff0d792e6ff53fe4daaf8aed5a3ba216669d2

**Issue description**:
When compiled using MSYS2, Synfig CLI tool fails to render `synfig-studio/images/about_icon.sif` (in synfig source).

Steps to reproduce:

1. Install MSYS2 environment and fetch Synfig sources
2. Install dependencies by running `./1-setup-windows-msys2.sh` from synfig's source dir
3. Compile Synfig by running `./2-build-debug.sh`
4. Run this command:
```
 ~/synfig.git/_debug/build/bin/synfig  ~/synfig.git/synfig-studio/images/about_icon.sif -o ~/file.png
```

The command will create `file.png` in your home dir, but Synfig CLI will hang and will not exit until you interrupt it manually.

This issue creates problems with building Synfig GUI, as it cannot build images in `synfig-studio/images` dir - it just hangs at particular step. For this reason building of images is currently disabled in build scripts for Windows case -
https://github.com/synfig/synfig/blob/dbcff0d792e6ff53fe4daaf8aed5a3ba216669d2/autobuild/build.sh#L263-L266

Also, it worth to note that some other files from `images` dir are rendered without a problem. For example, this one:
```
 ~/synfig.git/_debug/build/bin/synfig  ~/synfig.git/synfig-studio/images/action_add_to_set_icon.sif -o ~/file.png
```

This is what we shouldn't forget to do after fixing this issue:
- [ ] Remove [workaround in build script](https://github.com/synfig/synfig/blob/dbcff0d792e6ff53fe4daaf8aed5a3ba216669d2/autobuild/build.sh#L263)
- [ ] Remove notes about this issue from [build instructions](https://synfig-docs-dev.readthedocs.io/en/latest/common/building.html)

**Comment by rodolforg** [comment]
When I try it (yes, I finally can build on Windows), I get this warning message:
```
synfig(13736) [19:27:34] warning: Canvases still open!
synfig(13736) [19:27:34] warning: D:\msys64\home\Usuario\synfig.git\synfig-studio\images\about_icon.sif: count()=-17891602
```
It is just a warning but the count value is really strange.

**Comment by rodolforg** [comment]
SynfigStudio hangs too! (It can be noticed if you start app by command line)

**Comment by morevnaproject** [comment]
This is solved in b04c5d978605093.
